src/main/scala/viper/gobra/translator/encodings/interfaces/InterfaceEncoding.scala

```diff
@@ -686,7 +686,6 @@ class InterfaceEncoding extends LeafTypeEncoding {
     */
```

Comment:
would be good to extend the encoding's description in the line above here

Reply:
while you're on it, I would somehow indicate that there is one implication per implementation / type that implements the interface. This is not immediately clear in the current version of the encoding's description